builtin/settingtypes.txt

```diff
@@ -953,7 +953,7 @@ mapgen_limit (Map generation limit) int 31007 0 31007
 #    and jungle grass, in all other mapgens this flag controls all decorations.
 mg_flags (Mapgen flags) flags caves,dungeons,light,decorations,biomes,ores caves,dungeons,light,decorations,biomes,ores,nocaves,nodungeons,nolight,nodecorations,nobiomes,noores
 
-[*Biome API noise parameters]
+[*Biome API Noises]
```

Comment:
I don't think the plural form sounds right and what was wrong with the old one?

Reply:
The space is just enough (on 1920 × 1080, GUI scaling 1.0). It is located inside the navigation panel.

![Screenshot of Biome API settings](https://github.com/minetest/minetest/assets/4017302/e48625ec-30aa-4172-8d16-911fcb34f20a)

Reply:
I suggest "Biome API Nosie" then

Reply:
I think the category should just be called "Biome API", all the other mapgen categories also contain noise param settings. And each setting in the category already has "noise" in its name.